buildSrc/build.gradle

```diff
@@ -18,15 +18,30 @@
 
 repositories {
   mavenCentral()
+  maven { url "http://geode-maven.s3-website-us-west-2.amazonaws.com" }
 }
 
 dependencies {
   compile (group: 'org.apache.geode', name: 'geode-junit', version: '1.3.0') {
     exclude group: 'org.apache.logging.log4j'
   }
+  compile gradleApi()
+  compile 'org.apache.commons:commons-lang3:3.3.2'
+  compile 'org.apache.maven:maven-artifact:3.3.3'
+  compile 'com.github.docker-java:docker-java:3.1.2-GEODE'
 
   compile group: 'junit', name: 'junit', version: '4.12'
 
   testAnnotationProcessor this.project
+}
 
+sourceSets {
+  main {
+    java {
+      srcDirs = []
+    }
+    groovy {
+      srcDirs += ['src/main/java']
```

Comment:
Seems like a gradle/groovy gotcha. If your groovy code references java code then this needs to be in place since they are compiled separately. At least StackOverflow said so :)